test/e2e/kubevirt.go

```diff
@@ -353,24 +366,48 @@ var _ = Describe("Kubevirt Virtual Machines", func() {
 			}
 		}
 
-		startNorthSouthIngressIperfTraffic = func(containerName string, addresses []string, port int32, stage string) error {
+		startNorthSouthIperfTraffic = func(execFn func(cmd string) (string, error), addresses []string, port int32, logPrefix, stage string) error {
```

Comment:
nit: consider adding a type for the execFn attribute - `type execFn func(cmd string) (string, error)` 

not a must, but I think it pays off.